pkg/scd/operational_intents_handler.go

```diff
@@ -16,6 +16,42 @@ import (
 	"github.com/interuss/stacktrace"
 )
 
+// subscriptionCanBeRemoved will check if:
+// - a previous subscription was attached,
```

Comment:
Attached to what?  Neither this documentation nor the function signature mentions an operational intent.  I think we should add an op intent ID to the function signature (see other comment) and then this will make more sense (a previous subscription was attached to the specified operational intent).